### PR TITLE
Update vpp-agent to version 2.3.0

### LINKED ISSUE #1718: VPPAgent dataplane errors during testing
## Expected Behavior

Vpp agent should not disconnect

## Current Behavior

time="2019-10-04T11:00:46Z" level=error msg="rpc error: code = FailedPrecondition desc = KeyErrors: [config/vpp/v2/interfaces/DST-d9 (CREATE): not connected to VPP, ignoring the request, config/vpp/v2/interfaces/SRC-d9 (CREATE): not connected to VPP, ignoring the request]

and so on.

## Steps to Reproduce

Run following testcase

```go
// +build bench

package nsmd_integration_tests

import (
	"fmt"
	"sync"
	"testing"

	. "github.com/onsi/gomega"
	v1 "k8s.io/api/core/v1"

	"github.com/networkservicemesh/networkservicemesh/test/kubetest"
	"github.com/networkservicemesh/networkservicemesh/test/kubetest/pods"
)

func TestHugeConnectionCount(t *testing.T) {
	g := NewWithT(t)
	nodesCount := 2

	k8s, err := kubetest.NewK8s(g, true)
	defer k8s.Cleanup()
	//defer kubetest.MakeLogsSnapshot(k8s, t)
	g.Expect(err).To(BeNil())

	config := []*pods.NSMgrPodConfig{}
	for i := 0; i < nodesCount; i++ {
		cfg := &pods.NSMgrPodConfig{
			Variables: pods.DefaultNSMD(),
		}
		cfg.Namespace = k8s.GetK8sNamespace()
		cfg.DataplaneVariables = kubetest.DefaultDataplaneVariables(k8s.GetForwardingPlane())
		config = append(config, cfg)
	}
	nodes_setup, err := kubetest.SetupNodesConfig(k8s, nodesCount, defaultTimeout, config, k8s.GetK8sNamespace())
	g.Expect(err).To(BeNil())

	// Run ICMP on latest node
	_ = kubetest.DeployICMP(k8s, nodes_setup[nodesCount-1].Node, "icmp-responder-nse-1", defaultTimeout)

	var wg sync.WaitGroup
	count := 1000
	cycle := 5
	for j := 0; j < cycle; j++ {
		wg.Add(1)
		go func(j int) {
			defer wg.Done()
			for i := 0; i < count; i++ {
				var nscPodNode *v1.Pod
				podName := fmt.Sprintf("nsc-%d-%d", j, i)
				//logrus.Infof("podName=%v", podName)
				nscPodNode = kubetest.DeployNSC(k8s, nodes_setup[i%len(nodes_setup)].Node, podName, defaultTimeout)

				kubetest.CheckNSC(k8s, nscPodNode)
				k8s.DeletePods(nscPodNode)
			}
		}(j)
	}
	wg.Wait()
}

```
## Context

* Kubernetes Version:
* etc.

### PR DESCRIPTION
**Pull request by artem-belov**
Signed-off-by: Artem Belov <artem.belov@xored.com>

## Description
Update vpp-agent version
Update supervisord.conf

## Motivation and Context
New vpp-agent resolves #1718 

## How Has This Been Tested?

- [x] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by denis-tingajkin** [comment]
It looks like, with a new vppagent version something changed in metrics staff.
@artem-belov , @edwarnicke, Can we disable `TestSimpleMetrics` for a while and create an Issue to solve this in separate PR?

**Comment by artem-belov** [comment]
@denis-tingajkin  I have disabled TestSimpleMetrics. Could you please take a look what is wrong with it.

**Comment by denis-tingajkin** [comment]
@artem-belov ok, I'll try to fix https://github.com/networkservicemesh/networkservicemesh/issues/1763 in separate PR.

File: test/integration/basic_monitor_crossconnect_metrics_test.go [hunk 1]
```diff
@@ -18,6 +18,12 @@ import (
 
 func TestSimpleMetrics(t *testing.T) {
 	g := NewWithT(t)
+
+	if !kubetest.IsBrokeTestsEnabled() {
+		t.Skip("Temporary skip TestSimpleMetrics with vpp-agent v2.3.0")
+		return
+	}
+
```
Comment (edwarnicke) [inline test/integration/basic_monitor_crossconnect_metrics_test.go]: Why are we skipping this test?

Comment (haiodo) [reply test/integration/basic_monitor_crossconnect_metrics_test.go]: It seems Artyom is waiting for Denis feedback on this test.

Comment (edwarnicke) [reply test/integration/basic_monitor_crossconnect_metrics_test.go]: Nevermind... I see now.  And see we are on it.

**Pull request merged**